### DISCUSSION TITLE
monitoring binaries for uMurmur

### DISCUSSION THREAD

**Discussion by hw-lunemann** [Freetz-NG/freetz-ng#372] [Unerfüllte Wünsche]
uMurmur has two official tools for monitoring the server: umurmur-monitor and nuMurmon.
Both of them have their own repositories but can also cloned recursively from the main uMurmur repo.
Building them requires to compile uMurmur with --enable-shmapi (autotools) or --DUSE_SHAREDMEMORY_API=On (cmake).

I tried to look into setting that up but I can't really wrap my head around it between requiring a specific flag for uMurmur, different build systems (uMurmur is still being built with autotools which has been deprecated by uMurmur) and the monitor binaries themselves requiring uMurmur as they would be useless otherwise.

**Comment by fda77** [comment]
cmake is not supported in freetz, thats also the reason why getdns cant be updated. Someone needs to add it
